zingolib/src/wallet/transaction_records_by_id.rs

```diff
@@ -20,9 +30,108 @@ impl std::ops::DerefMut for TransactionRecordsById {
         &mut self.0
     }
 }
+
+/// This block implements constructors.
+impl Default for TransactionRecordsById {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
 impl TransactionRecordsById {
+    // New empty Map
+    pub fn new() -> Self {
+        TransactionRecordsById(HashMap::new())
+    }
     // Associated function to create a TransactionRecordMap from a HashMap
     pub fn from_map(map: HashMap<TxId, TransactionRecord>) -> Self {
         TransactionRecordsById(map)
     }
 }
+/// This block implements methods to modify the map.
+impl TransactionRecordsById {
+    /// All information after a certain height is invalidated during a reorg.
```

Comment:
add function description above these extra details. also when talking about height I think we should use `above` or `higher` instead of `after`
```suggestion
    /// Invalidates all transactions from a given height including the block with block height `reorg_height`
    ///
    /// All information above a certain height is invalidated during a reorg.
```